src/Layout/Page/Page.spec.tsx

```diff
@@ -2,17 +2,15 @@
 import * as React from 'react';
 import * as renderer from 'react-test-renderer';
 // VENDOR
-import { shallow, mount } from 'enzyme';
+import { shallow } from 'enzyme';
 // COMPONENT
 import {
     Page,
-    DEFAULT_HEADER_HEIGHT,
-    DEFAULT_FOOTER_HEIGHT,
 } from './Page.component';
 
 describe('Component: Page', () => {
     const subject = <Page />;
-    const wrapper = mount(subject);
+    // const wrapper = mount(subject);
```

Comment:
hmm

Reply:
👎

Reply:
Buttt, the tests that were here only tested the presence of the grid, which snapshots do so they weren't tests we needed anyway.